src/client_sync.rs

```diff
@@ -46,11 +46,13 @@ use crate::errors::{Error, Result};
 use crate::frame::{Frame, FrameOutput, FramedMessage};
 use crate::ADDRESS_SEP;
 use crate::client::jitter;
+use flume::Receiver;
+use flume::Sender;
```

Comment:
Keeping it consistent
```suggestion
use flume::{Receiver, Sender};
```